lib/rtcsessiondescription/index.js

```diff
@@ -1,17 +1,21 @@
+/* globals RTCSessionDescription */
 'use strict';
 
-var guessBrowser = require('../util').guessBrowser;
-
-switch (guessBrowser()) {
-  case 'chrome':
-    module.exports = require('./chrome');
-    break;
-  case 'firefox':
-    module.exports = require('./firefox');
-    break;
-  default:
-    if (typeof RTCSessionDescription === 'undefined') {
+if (typeof RTCSessionDescription === 'function') {
```

Comment:
we should all this checks for webRTC related types into isWebRTCSupported and use it here.

Reply:
@makarandp0 Since this module depends on a specific API, its OK to check on only that API.